main.go

```diff
@@ -145,10 +146,12 @@ func findArtifact(rootDir, ext string, buildStart time.Time) ([]string, error) {
 	return matches, nil
 }
 
-func checkBuildProducts(apks []string, apps []string, ipas []string, platforms []string, target string) error {
+func checkBuildProducts(apks []string, aabs []string, apps []string, ipas []string, platforms []string, target string) error {
 	// if android in platforms
-	if len(apks) == 0 && sliceutil.IsStringInSlice("android", platforms) {
-		return errors.New("No apk generated")
+	if sliceutil.IsStringInSlice("android", platforms) {
+		if len(apks) == 0 || len(aabs) == 0 {
```

Comment:
This will be true if only apk or aab is generated. But the issue is if none of the is generated: 
`if len(apks) == 0 && len(aabs) == 0 {`

Reply:
a new test case would be great:

```
		 	"Android aab generated",
 			args{
 				[]string{},
 				[]string{"/path.aab"},
 				[]string{},
 				[]string{},
 				[]string{"android"},
 				"device",
 			},
```